test-cases/PR-provision-test-docker.yaml

```diff
@@ -0,0 +1,15 @@
+test_duration: 15
```

Comment:
why do we have special YAML file for Docker?

Reply:
I've started like that cause of `use_mgmt: true` that wasn't test fully (and take a longer time to run and install)
and scylla-bench isn't install to the docker loader version.

I can do both in a different PR and remove this file afterwards